WeakAuras/RegionTypes/Icon.lua

```diff
@@ -223,7 +235,8 @@ local function create(parent, data)
   local cooldown = CreateFrame("COOLDOWN", "WeakAurasCooldown"..frameId, region, "CooldownFrameTemplate");
   region.cooldown = cooldown;
   cooldown:SetAllPoints(icon);
-  cooldown:SetDrawEdge(false);
+  region:SetCooldownEdge(data.cooldownEdge);
+  region:SetCooldownSwipe(data.cooldownSwipe);
```

Comment:
but actually this doesn't work -- the region methods are defined in modify() and not callable here.

Reply:
changed create() back to using SetDrawEdge/SetDrawSwipe for this reason.

if you want to move the region function definitions outside of modify(), that's probably best done separately from this pr...

Reply:
Oh, I'm an idiot. I didn't notice that was the create method.

Reply:
So, yeah, it kind of would be nice to change how exactly modify() works. But that's definitely beyond the scope of this PR. 😂